### ISSUE TITLE
How to change the width of a dropdown

### ISSUE THREAD

**Issue by emexal-company** [adobe/spectrum-web-components#1155]
I tried to set the css width of the `<sp-dropdown>`, but the size of the component doesn't change. Any idea how to acheive this ?

**Comment by adixon-adobe** [comment]
I can see that there's a css variable controlling this, so e.g. `--spectrum-picker-width: 500px;` would override to 500px wide. I'd make sure you double-check that on updates, since the variable names have changed a bit more often than the component APIs.

@Westbrook I don't see that in the dropdown API though: https://opensource.adobe.com/spectrum-web-components/components/dropdown/api -- it this something you're already working through with another issue?

**Comment by Westbrook** [comment]
Looking at this and its structure, it likely _never_ worked 🤕 My appologies.

The use of `--spectrum-picker-width: 500px;` also won't currently work because of the middle state of where we are for t-shirt sizing. I'll take a closer look at what we'll need to do here to get width control on the menu for this element!

**Comment by hunterloftis** [comment]
@emexal-company this is the hack I've been using to get around this while t-shirt sizes are in progress. Just be sure to set it as specifically as possible to avoid messing with dimensions elsewhere:

```
        sp-picker {
            --spectrum-global-dimension-size-2400: 100%;
        }
```

The `#button` in `sp-picker` is sized by `--spectrum-picker-width`, which it derives from `--spectrum-global-dimension-size-2400`. So overriding that value can let you indirectly pass new widths in.

**Comment by Westbrook** [comment]
Best to use capital letters in the word _HACK_ there. Futzing with pseudo-private APIs breaks a lot of stability guarantees, etc., etc. The fix in #1232 should get this back to customizable in a more common/sustainable way. Feel free to share you thoughts therein, if you see any concepts you'd like covered that are not. Current target is to have that in a release (possibly Thursday) this week...
